tests/Doctrine/Tests/ORM/Query/QueryTest.php

```diff
@@ -583,9 +584,17 @@ private function createConnection(Driver\Result ...$results): Connection
         $driverConnection->method('query')
             ->will($this->onConsecutiveCalls(...$results));
 
+        $platform = $this->getMockBuilder(AbstractPlatform::class)
+            ->setMethods(['supportsIdentityColumns'])
```

Comment:
```suggestion
            ->onlyMethods(['supportsIdentityColumns'])
```

`setMethods()` is deprecated. Let's avoid using it if we can.